packages/kotti-ui/source/kotti-field/hooks.ts

```diff
@@ -9,6 +9,7 @@ import {
 	Ref,
 } from '@vue/composition-api'
 import cloneDeep from 'lodash/cloneDeep'
+import shortid from 'shortid'
```

Comment:
1) this is deprecated https://www.npmjs.com/package/shortid

Reply:
2. i think it's much simpler to use a sequential incremental approach (there's an existing hook used for formIds for example)